tests/acc_test.go

```diff
@@ -468,80 +468,89 @@ func TestTerraformAwsModulesVpcIntoTypeScript(t *testing.T) {
 		pulumiConvert(t, localProviderBinPath, pclDir, testDir, "typescript", generateOnly)
 	})
 
-	pt := newPulumiTest(t, testDir,
-		opttest.LocalProviderPath(provider, filepath.Dir(localProviderBinPath)),
-		opttest.SkipInstall())
-	pt.CopyToTempDir(t)
+	for _, executor := range []string{"terraform", "tofu"} {
```

Comment:
These mini-matrices are super pesky to push into every test. 

Do you think we could maybe more elegantly solve it at the test automation layer by for example making a matrixed job over here that runs all the tests for several distinct non-default test configurations:

https://github.com/pulumi/pulumi-terraform-module/pull/362/files#diff-1cc0f4593a580c68638dfa17ba8d112c06d2281f89355e20a1325e9e8dea4ca1R64

Such as terraform+views, tofu+views, tofu.